plugins/modules/aci_dns_profile.py

```diff
@@ -224,9 +275,35 @@ def main():
     aci.get_existing()
 
     if state == "present":
+
+        child_configs = []
+        if management_epg is not None:
+            if management_epg == "":
+                child_configs.append(
+                    dict(
+                        dnsRsProfileToEpg=dict(
+                            attributes=dict(
+                                tDn="",
+                                status="deleted",
+                            )
+                        )
+                    )
+                )
+            else:
+                child_configs.append(
+                    dict(
+                        dnsRsProfileToEpg=dict(
+                            attributes=dict(
+                                tDn="uni/tn-mgmt/mgmtp-default/{0}-{1}".format(management_epg_type, management_epg),
```

Comment:
Shall we add the following, seeing that both management_epg_type and management_epg are required in the tDn?
```
required_together=[
  ('management_epg', 'management_epg_type'),
],
```

Reply:
I think this does not need to be because removing is "", and then you do not need the type. We could add a separate validation for it when the management_epg is provided but is not "".

Reply:
Yeah, a separate validation would be good but not necessary I guess. Your call!

Reply:
Is this validation required when the default for `management_epg_type` is `ooband`?

Reply:
no not required indeed, this is why I set default